005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,194 @@
+Summing the net total of all the initial responses to zero, allows us to explore the oscillatory nature of emotion while avoiding the possible pitfall of any dominant emotions. \par
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{@{}lllll@{}}
+    \toprule
+    HP dec last tick [nonfriend]    & Stubbornness      & Morality         & Mood             & Friendship     \\ \midrule
+    ask Food Taken                  & 0                 & -0.5             & -0.5             & 0              \\
+    ask HP                          & 0                 & -0.5             & -0.5             & 0              \\
+    ask Intended Food Intake        & 0                 & 0                & 0                & 0              \\
+    Request Take Food               & 1                 & -0.5             & -1               & -1             \\
+    Request Leave Food              & 1                 & -0.5             & -1               & 0              \\ \bottomrule
+    \end{tabular}
+\end{table}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{@{}lllll@{}}
+    \toprule
+    HP inc last tick [nonfriend]    & Stubbornness      & Morality         & Mood             & Friendship     \\ \midrule
+    ask Food Taken                  & 0                 & 0                & 0                & -1             \\
+    ask HP                          & -1                & 1                & 1.5              & 1              \\
+    ask Intended Food Intake        & 0                 & 0                & 0                & 0              \\
+    Request Take Food               & 0                 & 0                & 0                & 0              \\
+    Request Leave Food              & -1                & 1                & 1.5              & 1              \\ \bottomrule
+    \end{tabular}
+\end{table}
+\begin{table}[htb]
+    \centering
+    \begin{tabular}{@{}lllll@{}}
+    \toprule
+    Sum total                       & Stubbornness      & Morality         & Mood             & Friendship     \\ \midrule
+    Request Leave Food              & 0                 & 0                & 0                & 0              \\ \bottomrule
+    \end{tabular}
+\end{table}
+This zero-sum agent is how our agent is currently set, it represents the neutral agent. One who possesses no inherent emotional bias. This was done by balancing the positive and negative reactions to a friend, and those to a non-friend separately. We noticed the need for this when our agent would initially get stuck at the +ve and -ve limits of certain emotions. As future work, it would be exciting to see the emergent strategies which we hope to observe as we introduce biases. For instance, an agent whose net sum of all interactions leaves them with a stubbornness increase, coupled with an inherently moral agent, may create strategies yet thought of. 
```

Comment:
```suggestion
This zero-sum agent is how the agent is currently set, it represents the neutral agent. One who possesses no inherent emotional bias. This was done by balancing the positive and negative reactions to a friend, and those to a non-friend separately. We noticed the need for this when the agent would initially get stuck at the positive and negative limits of certain emotions. As future work, it would be exciting to see the emergent strategies which we hope to observe as we introduce biases. For instance, an agent whose net sum of all interactions leaves them with a stubbornness increase, coupled with an inherently moral agent, may create strategies yet thought of. 
```